src/services/mover/savings/service.ts

```diff
@@ -71,11 +72,19 @@ export const getSavingsReceipt = async (
     }
 
     const payloadProcessed: SavingsReceipt = {
-      ...response.payload,
-      hourlyBalances: response.payload.hourlyBalances.map((item) => ({
-        ...item,
-        type: 'savings_hourly_balance_item'
-      }))
+      avgDailyEarnings: response.payload.avgDailyEarnings,
+      earnedThisMonth: response.payload.earnedThisMonth,
+      endOfMonthBalance: response.payload.endOfMonthBalance,
+      paidToTreasury: response.payload.paidToTreasury,
+      savedFees: response.payload.savedFees,
+      totalDeposits: response.payload.totalDeposits,
+      totalWithdrawals: response.payload.totalWithdrawals,
+      hourlyBalances: isFeatureEnabled('isSavingsMonthlyChartEnabled')
+        ? response.payload.hourlyBalances!.map((item) => ({
```

Comment:
```suggestion
        ? response.payload.hourlyBalances?.map((item) => ({
```
I think this will be better than using non-null assertions

This value isn't something we are sure is always present so it's better to assume hourlyBalances may be undefined